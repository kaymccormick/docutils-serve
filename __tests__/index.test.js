const middleware = require('../index');


test('index', async () => {
    const next = () => {};
console.log(middleware);
    let markup;
    const render = (markup2) => {
	console.log(markup2);
	    expect(markup2).toMatchSnapshot();
    };
    const result =  await middleware({ path: '/foo', next, render });
    return result;
});
