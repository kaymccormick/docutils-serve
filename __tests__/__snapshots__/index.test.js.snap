// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index 1`] = `undefined`;
